frontend/src/text_resources.js

```diff
@@ -197,6 +197,8 @@ let LOCALIZE = new LocalizedStrings({
 
       //Test tabs
       tabs: {
+        menuName: "emib-tab-menu",
```

Comment:
Are these CSS classnames or code only? If so then they shouldn't be in translations. 

If they are meant to be read by a person, don't use dashes.

Reply:
fixed in #114 (commit 58ee3e9 )